itl/command-plugins.conf

```diff
@@ -557,6 +557,206 @@ object CheckCommand "http" {
 	vars.http_verbose = false
 }
 
+object CheckCommand "curl" {
+	import "ipv4-or-ipv6"
+
+	command = [ PluginDir + "/check_curl" ]
+
+	arguments += {
+		"-H" = {
+			value = "$curl_vhost$"
+			description = "Host name argument for servers using host headers (virtual host). Append a port to include it in the header (eg: example.com:5000)"
+		}
+		"-I" = {
+			value = "$curl_ip$"
+			set_if = {{ string(macro("$curl_ip$")) != "" }}
+			description = "IP address or name (use numeric address if possible to bypass DNS lookup)."
+		}
+		"-p" = {
+			value = "$curl_port$"
+			description = "Port number (default: 80)"
+		}
+		"-4" = {
+			set_if = "$curl_ipv4$"
+			description = "Force `check_curl` to use IPv4 instead of choosing automatically"
+		}
+		"-6" = {
+			set_if = "$curl_ipv6$"
+			description = "Force `check_curl` to use IPv6 instead of choosing automatically"
+		}
+		"(-S w/ value)" = {
+		  set_if = {{ macro("$curl_tls$") && string(macro("$curl_tls_version$")) != "" }}
+		  key = "-S"
+		  value = "$curl_tls_version$"
+		  description = "Connect via SSL. Port defaults to 443. VERSION is optional, and prevents auto-negotiation"
+		}
+		"(-S w/o value)" = {
+		  set_if = {{ macro("$curl_tls$") && string(macro("$curl_tls_version$")) == "" }}
+		  key = "-S"
+		  description = "Connect via SSL. Port defaults to 443. VERSION is optional, and prevents auto-negotiation"
+		}
```

Comment:
The indentation of this block differs from the rest. While everything else from your `CheckCommand` is indented with tabs, this has spaces.

Unfortunately, I was unable to use the GitHub suggestion thingy for whitespaces. Thus, I have attached a diff. However, you might just wanna use your local editor as I have a feeling that the GitHub web view will mess up whitespaces again.

```diff
diff --git a/itl/command-plugins.conf b/itl/command-plugins.conf
index 612d63f86..80a757fd2 100644
--- a/itl/command-plugins.conf
+++ b/itl/command-plugins.conf
@@ -585,15 +585,15 @@ object CheckCommand "curl" {
                        description = "Force `check_curl` to use IPv6 instead of choosing automatically"
                }
                "(-S w/ value)" = {
-                 set_if = {{ macro("$curl_tls$") && string(macro("$curl_tls_version$")) != "" }}
-                 key = "-S"
-                 value = "$curl_tls_version$"
-                 description = "Connect via SSL. Port defaults to 443. VERSION is optional, and prevents auto-negotiation"
+                       set_if = {{ macro("$curl_tls$") && string(macro("$curl_tls_version$")) != "" }}
+                       key = "-S"
+                       value = "$curl_tls_version$"
+                       description = "Connect via SSL. Port defaults to 443. VERSION is optional, and prevents auto-negotiation"
                }
                "(-S w/o value)" = {
-                 set_if = {{ macro("$curl_tls$") && string(macro("$curl_tls_version$")) == "" }}
-                 key = "-S"
-                 description = "Connect via SSL. Port defaults to 443. VERSION is optional, and prevents auto-negotiation"
+                       set_if = {{ macro("$curl_tls$") && string(macro("$curl_tls_version$")) == "" }}
+                       key = "-S"
+                       description = "Connect via SSL. Port defaults to 443. VERSION is optional, and prevents auto-negotiation"
                }
                "--sni" = {
                        set_if = "$curl_sni$"
```

Btw, there is a mixed indentation for the whole file - at least my vim complains about `mix-indent-file` -, but I would like to have consistent blocks, making a potential future fix more readable.

Reply:
oversight on my side. fixed it